modules/functions.js

```diff
@@ -37,6 +37,7 @@ module.exports = (client) => {
   // getSettings merges the client defaults with the guild settings. guild settings in
   // enmap should only have *unique* overrides that are different from defaults.
   client.getSettings = (guild) => {
+    client.settings.ensure('default', client.config.defaultSettings)
```

Comment:
alternitavly 
```js
client.settings.ensure('default', client.config.defaultSettings)``` can be used

Reply:
Aren't those 2 statements exactly identical? Also, client.config.defaultSettings doesn't exist when using setup.js - this needs to be clarified, I did not correctly update that myself.